sql/sql_prepare.cc

```diff
@@ -4149,25 +4149,6 @@ bool Prepared_statement::execute(String *expanded_query, bool open_cursor)
       thd->protocol->send_out_parameters(&this->lex->param_list);
   }
 
-  /*
-    Log COM_STMT_EXECUTE to the general log. Note, that in case of SQL
-    prepared statements this causes two records to be output:
-
-    Query       EXECUTE <statement name>
-    Execute     <statement SQL text>
-
-    This is considered user-friendly, since in the
-    second log entry we output values of parameter markers.
-
-    Rewriting/password obfuscation:
-
-    - Any passwords in the "Execute" line should be substituted with
-      their hashes, or a notice.
-
-  */
-  if (error == 0)
-    log_execute_line(thd);
```

Comment:
Why this change?

Reply:
We have exactly the same comment and call to log_execute_line(thd); as well.
This is consequence of bad merge done in commit 520b939d1b86b8cd5c906fb7f1711b96555f12f8 where PS commit 195d638f08eba9f797c19f4f33b220546c52081e was merged into PXC. 

PS change was introduced together with merging of MySQL upstream 5.6.24 in commit 195d638f08eba9f797c19f4f33b220546c52081e

This was discovered by failing mtr test and fixed.